src/becca/entities/rows.ts

```diff
@@ -69,19 +69,20 @@ export interface AttributeRow {
     noteId: string;
     type: AttributeType;
     name: string;
-    position: number;
-    value: string;
-    isInheritable: boolean;
+    position?: number;
+    value?: string;
+    isInheritable?: boolean;
     utcDateModified?: string;
 }
 
 export interface BranchRow {
     branchId?: string;
     noteId: string;
     parentNoteId: string;
-    prefix: string | null;
-    notePosition: number;
-    isExpanded: boolean;
+    prefix?: string | null;
+    notePosition: number | null;
```

Comment:
@Micosilent , maybe I misunderstand something but that does not seem the case.

If we have `foo?: number` then the type of `foo` will be `number | undefined`, not `number | null`. Uncommenting this would cause build errors where `null` is used such as in:

```
src/becca/entities/bbranch.ts:281:17 - error TS2322: Type 'string | null' is not assignable to type 'string | undefined'.
  Type 'null' is not assignable to type 'string | undefined'.
```

Reply:
You are right, I am not used to strict null checks.